pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   dartz: ^0.10.0-nullsafety.1
   kahootify_server:
     path: packages/server
+  numberpicker: ^2.1.1
```

Comment:
move it above kahootify_server. That's remote package and it's place is among it's kind

Reply:
ready